### ISSUE TITLE
fast forward

### ISSUE THREAD

**Issue by puba** [PopcornTimeTV/PopcornTimeTV#184]
Got strange fast forward issue on ver.070, every media content loaded is automatically fast forwarded to random time position and therefore can not be viewed??

**Comment by tiferrei** [comment]
Thanks for reporting @puba, may I ask which tvOS version are you running?

Tiago

**Comment by puba** [comment]
Latest official release, seems like the software is updating automatically, although I have switched automatically updating settings to off....odd things indeed

**Comment by tiferrei** [comment]
Hum that is odd indeed, most support cases would tell you to restore it, since the OS is misbehaving, but I don't believe in the Potato logic, so I'll investigate this issue for a bit and come back with a summary of what I think it might be, is that alright?

**Comment by puba** [comment]
Sounds i.e reads pretty good too me....such strange behaviour appears for example at last episodes of Powerpuff girls and The Walking dead series.

**Comment by puba** [comment]
Still experiencing issues on latest 361 build, media content is stopped after a few seconds of playing, then skipped to the end timeline and stops on one scenario, on other it just jumps forward for a few minutes and stalls.

**Comment by tiferrei** [comment]
Hum, just randomly or does that happen in a specific time or movie/episode?

**Comment by puba** [comment]
On Powerpuff girls series I got the first scenario, from first till last  episode....On Sillicon Valley series the other scenario, random skipping few minutes ahead. On older series everything seems to work fine. Is this somehow connected with newest video codecs?

**Comment by tiferrei** [comment]
Don't believe so, but what triggers it? Is it a specific time when it happens, or do you do something?

**Comment by puba** [comment]
No, it is not codecs, more subtitles related. If media is loaded without subs, it runs fine, if it is (pre)loaded with some subs, here comes the time skipping.

**Comment by tiferrei** [comment]
Hum, thanks I'll take a look at that

**Comment by puba** [comment]
Really hard to tell, what and why is wrong, because some media plays just fine, while other just don't, for example different episodes of same series. Guess thats the torrent price we got to pay for our free popcorn time.

**Comment by tiferrei** [comment]
@puba, thank you for waiting, we are analysing the issue and it now our top priority, as it seems to affect a lot of people, we believe this is related to the Popcorn Torrent Client and a patch is on its way.

**Comment by puba** [comment]
@tiferrei, thanks, much appreciated.
